public/index.ts

```diff
@@ -1097,40 +1105,33 @@ function goToZSlice(slice: number): boolean {
 function createTestVolume() {
   /* eslint-disable @typescript-eslint/naming-convention */
```

Comment:
Noted a couple of these left over, might be worth doing a pass through the whole repo to search for any that can be removed